_includes/configuration/format-empty-lines.md

```diff
@@ -0,0 +1,18 @@
+## `format_empty_lines`
+
+{{ site.requires_5_6v }}
+
+This option allows "inline" formats to be applied to empty lines for multi-line selections. Inline formats include bold (`<strong>`) and underline (`<span style="text-decoration: underline;">`).
```

Comment:
Are these the only two inline formats that can be applied? If not, is it worth adding the entire set?